cvat/apps/dataset_manager/formats/openimages.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import glob
+import os.path as osp
+from tempfile import TemporaryDirectory
+
+from datumaro.components.dataset import Dataset, DatasetItem
+from datumaro.plugins.open_images_format import OpenImagesPath
+from datumaro.util.image import DEFAULT_IMAGE_META_FILE_NAME
+from pyunpack import Archive
+
+from cvat.apps.dataset_manager.bindings import (GetCVATDataExtractor,
+    find_dataset_root, import_dm_annotations, match_dm_item)
+from cvat.apps.dataset_manager.util import make_zip_archive
+
+from .registry import dm_env, exporter, importer
+
+
+def find_item_ids(path):
+    image_desc_patterns = (
+        OpenImagesPath.FULL_IMAGE_DESCRIPTION_FILE_NAME,
+        *OpenImagesPath.SUBSET_IMAGE_DESCRIPTION_FILE_PATTERNS
+    )
+
+    image_desc_patterns = (
+        osp.join(path, OpenImagesPath.ANNOTATIONS_DIR, pattern)
+        for pattern in image_desc_patterns
+    )
+
+    for pattern in image_desc_patterns:
+        for path in glob.glob(pattern):
+            with open(path, 'r') as desc:
+                next(desc)
+                for row in desc:
+                    yield row.split(',')[0]
+
+@exporter(name='Open Images', ext='ZIP', version='1.0')
```

Comment:
Maybe, we should be more specific about the version and add `v6`.

Reply:
Done.